pkg/cmd/trades.go

```diff
@@ -96,3 +92,77 @@ var tradesCmd = &cobra.Command{
 		return nil
 	},
 }
+
+// go run ./cmd/bbgo tradeupdate --session=ftx
+var tradeUpdateCmd = &cobra.Command{
```

Comment:
how about a `private` command to show both trade updates and order updates? because once your order is executed, you would get these updates (trades, orders, balances) ... etc